plugins/digitalocean/resoto_plugin_digitalocean/collector.py

```diff
@@ -522,6 +525,38 @@ def get_size(droplet: Json) -> Json:
             },
         )
 
+        neighbours = self.client.list_droplets_neighbors_ids()
+        neighbours_json = []
+        for neighbour in neighbours:
```

Comment:
naming - proposal for improvement:
neighbor -> neighbors (it is a list)
neighbors -> all_neighbor_lists

Reply:
Since it's called DigitalOceanDropletNeighbourhood we could also name it `for neighbourhood in neighbourhoods`.